provider/schema/column.go

```diff
@@ -164,6 +164,8 @@ type Column struct {
 	// IgnoreInTests if true this skips this column in tests as sometimes it might be hard
 	// to create a reproducible test environment with this column being non nill. For example various error columns and so on
 	IgnoreInTests bool
+	// Internal is true if this column is managed by the SDK
+	Internal bool
```

Comment:
unexported. Didn't notice this would only be used in schema, so it was left exported. If we move `ExecutionData` somewhere else we can always add a getter and keep this unwritable.